sys/random/prng_tinymt32.c

```diff
@@ -35,6 +36,21 @@ uint32_t random_uint32(void)
     return tinymt32_generate_uint32(&_random);
 }
 
+size_t random_bytes(uint8_t *buf, size_t size)
+{
+  size_t count = 0;
+  uint32_t last;
+
+  while(size > sizeof(uint32_t)) {
+    *((uint32_t*)(buf+count)) = tinymt32_generate_uint32(&_random);
```

Comment:
One way would be to round the buffer pointer up to nearest 32 bit word and perform the loop. Finally generate two more words of random and use them to fill the leading and trailing bytes of the buffer.